### PR TITLE
Added Hulks Robots and workshop venue

### LINKED ISSUE #534: Put information on workshop venue on the website
We're currently only saying the workshop venue is a different one in the city center. We should add the details, see https://www.notion.so/mainmatter/Venues-7c607961b70e465896a50e6df3b21dbe?pvs=4#5b80e8dd5073453fa68c412b32d4b39d

### LINKED ISSUE #558: Info about HULKs Robo-soccer team on conference days
Please help putting the information on both days of conference regarding HULKs Robo-soccer team exhibition. Please look at the notion page here: https://www.notion.so/mainmatter/Announce-http-hulks-de-robo-soccer-team-179ee00d55014b4abc2e1cb8fd592833?pvs=4

Photo to use:
240748037_6046847068720842_38655145853045758_n

Photo source: Photo by HULKs Team on instagram.com/hulks_tuhh/ 

Copy recommendation:
Title -> Meet the Rust-programmed robo-soccer team!
Body -> The HULKs RoboCup SPL Team is coming to EuroRust. Don’t miss their live demos, code showcases, and debugging tool sessions. 🤖

### PR DESCRIPTION
**Pull request by umbrellarocket**
1. Fixed error where breakfast appeared twice on mobile, once below "Main Stage" and once above it
2. Added the Hulks Robotics team to the beginning of the schedule for days 1 and 2
3. Added info about and link to the workshop venue

Re: #2, are you OK with the placement of the robot soccer team section right above the Breakfast entry for each day? I'm happy to change it if necessary
image

closes #534 
closed #558

### CODE REVIEW

File: templates/sections/schedule.html [hunk 1]
```diff
@@ -5,7 +5,9 @@ <h2 class="mb-4 h5" id="schedule">Schedule</h2>
     <div class="workshops mb-9">
     {% set workshops = load_data(path = "data/workshops.json") %}
       <h3 class="h3 mb-4">Pre Conference Day - Oct. 9th, 2024</h3>
-      <p class="mb-3">The EuroRust workshops will take place in a separate venue in the city center.</p>
+      <p class="mb-3">The EuroRust workshops will take place in WEXELERATE, located at Praterstrasse 1, 1020 Vienna, Austria.</p>
+      <a class="btn__secondary" href="https://maps.app.goo.gl/APG7SV8pm7qkycPV6" alt="Google Maps for workshop venue address"
+				target="_blank" rel="noopener noreferrer">Workshop Venue on the Map</a>
```
Comment (marcoow) [inline templates/sections/schedule.html:10]:
```suggestion
      <p class="mb-3">The EuroRust workshops will take place in WEXELERATE, located at <a class="btn__secondary" href="https://maps.app.goo.gl/APG7SV8pm7qkycPV6" alt="Google Maps for workshop venue address"
				target="_blank" rel="noopener noreferrer">Praterstrasse 1, 1020 Vienna, Austria.</a></p>
```

File: templates/sections/schedule.html [hunk 2]
```diff
@@ -48,11 +50,17 @@ <h3 class="h3 mb-4">Pre Conference Day - Oct. 9th, 2024</h3>
...
         <div class="mobile-schedule schedule__grid day-one mb-10">
-          <div class="schedule__shared mb-5">
-            <p class="large">9:00 - 10:00</p>
-            <p>Doors open & breakfast</p>
-          </div>
-          <div class="schedule__main-stage border-r">
+            <div class="closing-party closing-party__grid schedule__shared">
+              <div class="closing-party__content">
+                <h3 class="large mb-2">Meet the Rust-programmed robo-soccer team!</h3>
+                    <p>The HULKs RoboCup SPL Team is coming to EuroRust. Don’t miss their live demos, code showcases, and debugging tool sessions. 🤖</p>
```
Comment (marcoow) [inline templates/sections/schedule.html:56]:
```suggestion
                    <p>The HULKs RoboCup SPL Team is coming to EuroRust. They'll be showing their robots, code showcases, as well as debugging tools and more during both conference days. 🤖</p>
```

File: templates/sections/schedule.html [hunk 3]
```diff
@@ -116,6 +124,24 @@ <h3 class="large mb-2">18:00 - Closing Party</h3>
...
             {% set_global sideRow = 0 %}
             {% for activity in day.mainStage %}
+            {% if loop.first %}
+              <tr>
+                <td class="visually-hidden">18:00</td>
+                <td colspan="2">
+                  <div class="closing-party closing-party__grid schedule__shared">
+                    <div class="closing-party__content">
+                      <h3 class="large mb-2">Meet the Rust-programmed robo-soccer team!</h3>
+                      <p>The HULKs RoboCup SPL Team is coming to EuroRust. Don’t miss their live demos, code showcases, and debugging tool sessions. 🤖
```
Comment (marcoow) [inline templates/sections/schedule.html:134]:
```suggestion
<p>The HULKs RoboCup SPL Team is coming to EuroRust. They'll be showing their robots, code showcases, as well as debugging tools and more during both conference days. 🤖</p>
```

File: templates/sections/workshop-schedule.html [hunk 4]
```diff
@@ -4,7 +4,7 @@ <h2 class="h5 mb-4">Workshops</h2>
     <p class="h2 mb-4">Deepen your Rust skills hands-on!</p>
     <div class="workshops mb-9">
     {% set workshops = load_data(path = "data/workshops.json") %}
-      <p class="mb-5 mw-70">The EuroRust workshops will take place in a separate venue in the city center – details to be announced soon.</p>
+      <p class="mb-5 mw-70">The EuroRust workshops will take place in WEXELERATE, located at Praterstrasse 1, 1020 Vienna, Austria.</p>
```
Comment (marcoow) [inline templates/sections/workshop-schedule.html:7]: let's add a link here like above

File: templates/sections/workshop-venue.html [hunk 5]
```diff
@@ -8,7 +8,7 @@
 			<div class="venue__wrapper-content">
 				<h5 class="mb-4">Workshop Venue</h5>
 				<h2 class="mb-4">The place to learn</h2>
-				<p class="mb-8">The EuroRust workshops will take place in a separate venue in the city center – details to be announced soon.</p>
+				<p class="mb-8">The EuroRust workshops will take place in WEXELERATE, located at Praterstrasse 1, 1020 Vienna, Austria.</p>
```
Comment (marcoow) [inline templates/sections/workshop-venue.html:11]: let's add a link

**Review by marcoow — COMMENTED** [review]
On the workshop detail pages, let's add an image of the venue as we do for the main venue

**Comment by umbrellarocket** [comment]
@marcoow Thanks for the feedback! I added the links/picture as requested

**Review by marcoow — APPROVED** [review]

**Comment by marcoow** [comment]
I moved the robots to the end of the schedule – seems more logical

**Pull request merged**